subsys/bluetooth/shell/bt.c

```diff
@@ -1074,6 +1093,22 @@ static int cmd_scan_filter_set_addr(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int cmd_scan_filter_set_rssi(const struct shell *sh, size_t argc, char *argv[])
+{
+	int rssi = strtol(argv[1], NULL, 10);
+
+	if ((rssi < 50) && (rssi > -200)) {
```

Comment:
What's the rationale behind `50` and `-200`? 

The RSSI value report by the BT subsystem is a int8_t (thus can't be lower than `-128`).

Reply:
no rationale, really. I capped it to the int8_t max values